taker-electron/src/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Itchy Sats</title>
+    <style>
+        .logo {
+            animation: pulse 2s linear infinite;
+        }
+
+        body {
+            background-color: black;
+            margin: 0;
+            font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+            'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+            sans-serif;
+            -webkit-font-smoothing: antialiased;
+            -moz-osx-font-smoothing: grayscale;
+        }
+
+        .content {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            -webkit-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+        }
+
+        @-webkit-keyframes "pulse" {
+            0% {
+                -webkit-transform: scale(1.0);
+                transform: scale(1.0);
+            }
+            50% {
+                -webkit-transform: scale(0.8);
+                transform: scale(0.8);
+            }
+            100% {
+                -webkit-transform: scale(1);
+                transform: scale(1);
+            }
+        }
+
+        #message {
+            text-align: center;
+            color: white;
+        }
+    </style>
+    <script>
+        let eventSource = new EventSource("http://127.0.0.1:8000/api/feed", { withCredentials: true });
+        const eventType = "maker_status";
+        eventSource.addEventListener(eventType, (event) => {
+            const data = JSON.parse(event.data);
+            let message = document.getElementById("message");
+            if (data.online) {
+                // maker became online, Itchy Sats is ready.
+                message.innerText = "Ready!";
+                // close event source
+                eventSource.close();
+                eventSource.removeEventListener(eventType, this);
+                // push maker_status event from renderer to main process.
+                window.itchysats.load();
+            } else {
+                message.innerText = "Connecting to maker!";
+            }
+        });
+    </script>
+</head>
+<body>
+    <div class="content">
+        <img src="logo.svg" class="logo" />
+        <p id="message">Starting Itchy Sats ...</p>
+    </div>
+</body>
+</html>
```

Comment:
nit: no newline here

Reply:
Because of reasons :) https://stackoverflow.com/a/729795